docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,1021 @@
+:branch:          1.5
+:ls_version:		  1.5.0.beta1
+
+= How to write your own Logstash {plugintype} plugin
```

Comment:
I'd be inclined to streamline this to "How to write a Logstash {plugintype} plugin", but it's really just a style thing.